README.md

```diff
@@ -8,16 +8,19 @@ See [Docker's documentation](https://docs.docker.com/docker-hub/official_repos/)
 
 ## Architectures other than amd64?
 
-Work is in-progress in the Docker Engine and Registry to properly support multiple architectures (see [docker/docker#15866](https://github.com/docker/docker/issues/15866)). While this work is ongoing, **temporary**, experimental builds of the official images for the following architectures are happening somewhat incrementally (with a strong bias towards images necessary for Docker's own CI to help hasten proper multiarch support upstream) via CI.
+Some images have been ported for other architectures, and many of these are officially supported. These are the supported architectures:
 
--	ARMv5 (`armel`): https://hub.docker.com/u/armel/
--	ARMv7 (`armhf`): https://hub.docker.com/u/armhf/
--	ARMv8 (`arm64`): https://hub.docker.com/u/aarch64/
+-	IBM z Systems (`s390x`): https://hub.docker.com/u/s390x/
 -	POWER8 (`ppc64le`): https://hub.docker.com/u/ppc64le/
--	System z (`s390x`): https://hub.docker.com/u/s390x/
 -	x86/i686 (`i386`): https://hub.docker.com/u/i386/
+-	ARMv8 (`arm64v8`): https://hub.docker.com/u/arm64v8/
+-	ARMv7 (`arm32v7`): https://hub.docker.com/u/arm32v7/
+-	ARMv6 (`arm32v6`): https://hub.docker.com/u/arm32v6/ (Raspberry Pi 1, Raspberry Pi Zero)
```

Comment:
Ok, we'll leave it as is. The problem I am worried about here is that you have `arm32` whereas OCI has `arm`.

I don't think this is a big deal.